### DISCUSSION TITLE
Property ... does not exists on type Store

### DISCUSSION THREAD

**Discussion by marcelruiter** [vuejs/pinia#880] [Help and Questions]
I created a PiniaPlugin with some actions like this 

```
const MyPlugin= () => {
  return {
    testMe: () => {
      console.log('test me from pinia plugin');
    },
  };
};
```

on my main.ts file I added pinia

```
// use pinia
import { createPinia } from "pinia";
const pinia = createPinia();
pinia.use(MyPlugin);
```

on my components I use other pinia stores

```
import { useStore } from "../store";
const store = useStore();
```

when I call ```store.testMe()``` the function is handled oke, it all works without any problem. but the linter throws errors `Property testMe does not exists on type Store.

how can I avoid this error from linter?

**Comment by BenShelton** [comment]
I assume this is TypeScript throwing this error rather than a linter. If that's the case then you need to add the type definitions for the `testMe` property you've added. There's a guide on how to do this: https://pinia.esm.dev/core-concepts/plugins.html#typing-new-store-properties

In short, create a declaration file (for example `pinia.d.ts`) and then add the following:

```ts
import 'pinia'

declare module 'pinia' {
  export interface PiniaCustomProperties {
    testMe: () => void
  }
}
```

**Comment by p9-web** [comment]
Hi,
I'm having a similar problem using getActivePinia:

`
const model = 'someModelName';
const _store = getActivePinia()._s?.get(model);
`

Getting this Typescript error: 

`TS2339: Property '_s' does not exist on type 'Pinia'.`
